instrumentation/opentelemetry-instrumentation-dbapi/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Stop capturing query parameters by default
+  ([#156](https://github.com/open-telemetry/opentelemetry-python/pull/156))
```

Comment:
should this be updated to reference this PR now?

Reply:
for sure, thanks!